bobtemplates/plone/addon/buildout.cfg.bob

```diff
@@ -3,7 +3,4 @@
 # use this extend one of the buildout configuration:
 extends =
 # -*- mrbob: extra extends -*-
-#    test_plone43.cfg
-#    test_plone50.cfg
-#    test_plone51.cfg
-    test_plone52.cfg
+    test_plone{{{ plone.version_digits }}}.cfg
```

Comment:
i would keep it like it was, this is not changing too much, only bigger version numbers and some may want to use switch to different versions too.